src/DssSpell.t.sol

```diff
@@ -1,2316 +1,279 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Looks like the spell was copied over this test file.

Reply:
Oops, that must have happened while fixing the merge conflicts. Just fixed it.